app/views/hiring_staff/vacancies/_show.html.haml

```diff
@@ -16,7 +16,7 @@
           #{link_to t('jobs.view_public_link'), @vacancy.share_url, class: 'govuk-link', id: 'share-url', 'aria-label': t('jobs.view_public_link')}, or #{link_to t('jobs.copy_public_url'), '#', class: 'govuk-link copy-to-clipboard', 'data-clipboard-text': @vacancy.share_url, 'aria-label': t('jobs.copy_public_url')}
         .share-url-no-js.mb1
           %h2.govuk-heading-s.mt0 Share this job
-          #{link_to "#{t('jobs.view_public_link')} (#{@vacancy.share_url})", job_path(@vacancy), class: 'govuk-link', id: 'share-url'}
+          #{link_to "#{t('jobs.view_public_link')} (#{@vacancy.share_url})", job_path(@vacancy), class: 'govuk-link', id: 'share-url-no-js'}
```

Comment:
Is the `id` even valuable for these links?  If not might be easier to remove the `id`s?

Reply:
conversely, is there any harm in leaving the `id`s on the elements?

Reply:
Only that other developers will come along and wonder why this element has an ID, when it doesn't seem necessary 😛

Reply:
amended this in https://github.com/DFE-Digital/teacher-vacancy-service/pull/994/commits/a09302c03e19a75cd6abf7dadc523090fc6a5bf4